livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/_oai_api.py

```diff
@@ -64,13 +64,19 @@ def create_ai_function_info(
             inner_type = typing.get_args(arg_info.type)[0]
```

Comment:
fetching the first argument does not work for type definitions like `Union[None, str]` or `None | str`, where the None is passed as the first argument. The validation should verify if arguments lengths is between 1 and 2 and then take the first matched non-None-type argument.